### ISSUE TITLE
SD writing seems somehow broken with 2.0 on macOS

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-hotspot#315]
In GitLab by @Kelson42 on Nov 18, 2018, 17:02

[i267-1.txt](/uploads/0c4fc1add0097a1e1ed2d52fc291f092/i267-1.txt)

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 18, 2018, 17:05

@rgaudin It seems to restart over and over after 8%

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 18, 2018, 17:06

assigned to @rgaudin

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 18, 2018, 17:45

Hum, 2 possibilities here:

* Etcher and/or hardware problem.
* Output reading issue (as we display the log file)

You have the command in the log ; would you try it separately in the terminal to check?

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 18, 2018, 17:52

What is exactly the command line?
```
/usr/bin/osascript -e 'do shell script "\'/Volumes/Kiwix Hotspot 4/Kiwix Hotspot.app/Contents/MacOS/etcher-cli/etcher\' -c -y -u -d /dev/disk3 /Volumes/Samsung_T5/hotspot-2018_11_18-09_11_50.img 2>&1 >/var/folders/s7/frvr3wkd3_34_kkd353f70k40000gn/T/tmp692e3y5e.log" with administrator privileges'
```

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 18, 2018, 17:53

```
sudo /Volumes/Kiwix Hotspot\ 4/Kiwix Hotspot.app/Contents/MacOS/etcher-cli/etcher -c -y -u -d /dev/disk3 /Volumes/Samsung_T5/hotspot-2018_11_18-09_11_50.img
```

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 18, 2018, 18:09

@rgaudin This works fine.

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 21, 2018, 05:56

@rgaudin New attempt with a brand new SD card and fails again

[i267-2.txt](/uploads/9924df00b0fb57626ea3063ce1597adf/i267-2.txt)

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 21, 2018, 08:26

changed the description

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 21, 2018, 08:35

I think it doesn't fail and you think it is because we're just periodically displaying the full log file, which goes from 0% to the latest available.

I'll read and parse the output so we only display the last useful line.

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 21, 2018, 11:44

closed via commit b72056fd957017392d466e24893ec52f391a881f
